src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -841,9 +841,9 @@
     itself, ZooKeeper associates all the ids that correspond to a
     client with the clients connection. These ids are checked against
     the ACLs of znodes when a clients tries to access a node. ACLs are
-    made up of pairs of <emphasis>(scheme:expression,
+    made up of pairs of <emphasis>(scheme:id,
```

Comment:
Oh, I see. ID is meant to be the complete `scheme:expression`. Need more coffee...